KubeArmor/BPF/Makefile

```diff
@@ -6,17 +6,17 @@ all: kernel_headers libbpf syscall_test system_monitor.bpf.o system_monitor.cont
 .PHONY: kernel_headers
 kernel_headers:
 ifeq ($(BTF_SUPPORTED),1)
-	@echo "$(GREEN)Kernel BTF information found$(NC)"
-	@echo "Generating vmlinux.h for kernel $(KRNV_X).$(KRNV_Y).$(KRNV_Z)"
+	@echo -e "$(GREEN)Kernel BTF information found$(NC)"
```

Comment:
This will not work across all platforms.

If I run the above, I get:

```
❯ make
-e Kernel BTF information found
Generating vmlinux.h for kernel 5.11.0
make[1]: Entering directory '/home/rahul/rnd/knox/KubeArmor/KubeArmor/BPF/tests'
Compiling eBPF bytecode: checks/security_path_unlink.c ...
Checking syscall checks/security_path_unlink
[Success] Syscall kprobe__security_path_unlink attached
Enabling syscall security_path_unlink
make[1]: Leaving directory '/home/rahul/rnd/knox/KubeArmor/KubeArmor/BPF/tests'
```

Note "-e" getting printed. What we currently have is correct imo.

Reply:
:sweat_smile: Okay, What I read was that this enables backspaces interpretation across different shell's but seems like it's not consistent.